tests/openvino/native/data/reference_scales/IntegerModel_mixed.json

```diff
@@ -3877,10 +3877,10 @@
             ]
         ]
     },
-    "Gather_2/fq_output_0": {
+    "Gather_2_data/fq_output_0": {
         "input_low": 0.0,
-        "input_high": 0.997209906578064,
```

Comment:
I am not sure, but I didn't fully get why the value is changed.

Reply:
Previously, FQ was placed after the `Gather` layer. For now, it puts before.